examples/pna-example-exactmap-ternarymap.p4

```diff
@@ -0,0 +1,319 @@
+    }
+    table emap1 {
+        key = {
+            // The key fields are all of the fields of struct
+            // emap1_key, with the same names as the member field
+            // names in the declaration of struct emap1_key.  All have
+            // match_kind exact, because emap1 is ExactMap, not
+            // TernaryMap.
+            my_field: exact;
+        }
+        actions = { lookup; }
+        default_action = lookup(42);
+    }
+*/
+    ExactMap<emap1_key, bit<16>>(
+        size = 1024,
+        const_entries = (list<exactmap_const_entry_t<emap1_key, bit<16>>>) {
+            {{ 5}, 10},  // key my_field= 5, value 10
+            {{ 6}, 27},  // key my_field= 6, value 27
+            {{10},  2}   // key my_field=10, value  2
+        },
+        default_value = 42)  // default value returned for all other keys
+    emap1;
+
+/*
+    The control plane API for ExactMap emap2 is exactly the same as
+    for the following P4 table.
+
+    // The action parameters of action 'lookup' correspond to the
+    // fields of struct 'emap2_val', "exploded" out in order to use
+    // the types and names of the member fields of the struct.
+    action lookup (bit<4> val1, bit<12> val2) {
+        // body of action is irrelevant for control plane API generation
+    }
+    table emap2 {
+        key = {
+            // The key fields are all of the fields of struct
+            // emap2_key.
+            val1: exact;
+            val2: exact;
+        }
+        actions = { lookup; }
+        default_action = lookup({val1= 0, val2=  28});
+    }
+*/
+    ExactMap<emap2_key, emap2_val>(
+        size = 128,
+        initial_entries = (list<exactmap_initial_entry_t<emap2_key, emap2_val>>) {
+            // not const entry, key my_field=5, value {val1=10, val2=4}
+            {false, { 5}, {val1=10, val2=   4}},
+            // const entry, key my_field=6, value {val1=15, val2=4095}
+            { true, { 6}, {val1=15, val2=4095}},
+            // const entry, key my_field=10, value {val1=0, val2=28}
+            { true, {10}, {val1= 0, val2=  28}}
+        },
+        // default value returned for all other keys
+        default_value = {val1=9, val2=42})
+    emap2;
+
+/*
+    The control plane API for TernaryMap tmap1 is exactly the same as
+    for the following P4 table.
+
+    // Note that the name 'lookup_value' is a default action parameter
+    // name generated by the P4 compiler in this case, because the
+    // value type of tmap1 is bit<16>, with no name.
+    action lookup (bit<16> lookup_value) {
+        // body of action is irrelevant for control plane API generation
+    }
+    table tmap1 {
+        key = {
+            // The key fields are all of the fields of struct
+            // tmap1_key.
+            f1: ternary;
+            f2: ternary;
+        }
+        actions = { lookup; }
+        default_action = lookup(42);
+    }
+*/
+    TernaryMap<tmap1_key, bit<16>>(
```

Comment:
Add explicit comment that earlier const_entries members are high matching priority than later ones, just like for 'const entries' in P4 tables.

Reply:
Added such a comment in definition of const_entries for TernaryMap externin pna.p4 file.